apps/src/templates/progress/ProgressButton.jsx

```diff
@@ -132,8 +132,8 @@ const ProgressButton = React.createClass({
     const color = this.props.color || ButtonColor.orange;
     const size = this.props.size || ButtonSize.default;
 
-    if (!!href === !!onClick) {
-      throw new Error('Expect exactly one of href/onClick');
+    if (!href && !onClick) {
+      throw new Error('Expect at least one of href/onClick');
     }
```

Comment:
@Bjvanminnen Is this okay?  In our case we open the link in a new window and report the click with `onClick` simultaneously.